CHANGELOG.md

```diff
@@ -24,3 +24,70 @@ bump for bug fixes,
 # 1.0.0.4
 bumped version for float/double range bugfix
 
+# 1.2
+
+1. Breaking change which mostly maintains backwards compatibility.
+2. Faster by more x10 (depending on the type) - see below for benchmarks.
+3. Passes a large number of random number test suites:
+  * [dieharder](http://webhome.phy.duke.edu/~rgb/General/dieharder.php "venerable")
+  * [TestU01 (SmallCrush, Crush, BigCrush)](http://simul.iro.umontreal.ca/testu01/tu01.html "venerable")
+  * [PractRand](http://pracrand.sourceforge.net/ "active")
+  * [gjrand](http://gjrand.sourceforge.net/ "active")
+  * [rademacher-fpl](https://gitlab.com/christoph-conrads/rademacher-fpl/-/tree/master "active")
```

Comment:
Our "simple but fast" floating point number generation does not pass rademacher-fpls' test suite.

Reply:
This must have crept back in

Reply:
I am very suspicious about those tests in any event. I can't understand what he is testing (well Kolmogorov-Smirnov is obvious) without some mathematical explanation and I am disinclined to reverse engineer.